tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -1662,6 +1713,14 @@ int tracepoint__raw_syscalls__sys_enter(struct bpf_raw_tracepoint_args *ctx)
             bpf_map_update_elem(&new_pids_map, &pid, &pid, BPF_ANY);
         }
     }
+    else if (id == CONNECT_SYSCALL || id == BIND_SYSCALL || id == LISTEN_SYSCALL) {
+        u32 sockfd = args_tmp.args[0];
+        save_sockfd(sockfd, id);
+    }
+    else if (id == ACCEPT_SYSCALL || id == ACCEPT4_SYSCALL) {
```

Comment:
we need it because the key in the map 'sockfd_map' is determined using the event_id.
fot the accept case, we have two syscalls: 'accept', 'accept4' - but only one lsm_hook: 'security_socket_accept'.
so to avoid checking in 'security_socket_accept' which syscall triggered the lsm_hook, i use the same key in the map for both syscalls (the SYSCALL_ACCEPT one).

Reply:
But actually, why do we even need to use the id here? If we entered a syscall, it is not possible that another syscall will happen for the same tid - maybe we can use one event-id here for all?

Reply:
i removed it all together - now using only tgid.